test/widgets/inset_shadow.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:ui' as ui;
+import 'package:checks/checks.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:legacy_checks/legacy_checks.dart';
+import 'package:zulip/widgets/inset_shadow.dart';
+
+import '../flutter_checks.dart';
+
+void main() {
```

Comment:
This test doesn't actually run in `flutter test` (or `tools/check`) because the name doesn't end with `_test.dart` :slightly_smiling_face: